src/main/java/org/nrnb/gsoc/enrichment/ui/EnrichmentTableModel.java

```diff
@@ -237,6 +237,22 @@ public void filter(List<TermSource> sources) {
         fireTableDataChanged();
     }
 
+    public void filterByEvidenceCode(Set<String> evidenceCodes) {
+        List<CyRow> rows = cyTable.getAllRows();
+        Long[] rowArray = new Long[rows.size()];
+        int i = 0;
+        for (CyRow row : rows) {
+            // implement this again
```

Comment:
Could you please explain this comment?

Reply:
Actually, this section of code is motivated from https://github.com/cytoscape/enrichment-table-app/blob/fc75bc7c50c8bc49b2e7bd8b55d2b1b6d079f4f0/src/main/java/org/nrnb/gsoc/enrichment/ui/EnrichmentTableModel.java#L224-L238

So, I am not aware why it was present in the first place.